### PR TITLE
209 jacoco does not work

### PR DESCRIPTION
**Pull request by yurichervoniy**

### CODE REVIEW

File: src/test/java/com/a5lab/tabr/ApplicationTestBaseConfig.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import java.lang.annotation.Retention;
```
Comment (arufanov) [inline src/test/java/com/a5lab/tabr/ApplicationTestBaseConfig.java:6]: @yurichervoniy should we move this file to test/.../tabr/config?

File: src/test/java/com/a5lab/tabr/AbstractControllerTests.java [hunk 2]
```diff
@@ -1,18 +1,11 @@
 package com.a5lab.tabr;
```
Comment (arufanov) [inline src/test/java/com/a5lab/tabr/AbstractControllerTests.java:2]: @yurichervoniy should we move AbstractControllerTests and AbstractRepositoryTests to tabr/domain. Similar with AbstractAuditable. All controlelr & repository test will be located at domain.

**Review by arufanov — COMMENTED** [review]
@yurichervoniy lets discuss a few conversation

File: src/test/java/com/a5lab/tabr/ApplicationTestBaseConfig.java (hunk 1 above)
Comment (yurichervoniy) [reply src/test/java/com/a5lab/tabr/ApplicationTestBaseConfig.java:6]: Done

File: src/test/java/com/a5lab/tabr/AbstractControllerTests.java (hunk 2 above)
Comment (yurichervoniy) [reply src/test/java/com/a5lab/tabr/AbstractControllerTests.java:2]: Done

**Review by arufanov — APPROVED** [review]

**Pull request merged**